CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 #### Upcoming Changes
 
-* feat: add `arbitrary` feature to enable arbitrary derive in `Program` and `CairoRunConfig`
+* feat: add `arbitrary` feature to enable arbitrary derive in `Program` and `CairoRunConfig` [#1306](https://github.com/lambdaclass/cairo-vm/pull/1306) [#1330](https://github.com/lambdaclass/cairo-vm/pull/1330)
```

Comment:
As this is a testing-oriented feature maybe we could change the `feat` to `feat(fuzzing)` or something specific so its less confusing for end users